examples/main/modules.tf

```diff
@@ -38,6 +43,23 @@ module "network_security_group" {
 
   # You can set either a prefix for generated name or a custom one for the resource naming
   #custom_network_security_group_names = "my_nsg"
+
+  # You can set either a prefix for generated name or a custom one for the resource naming
+  custom_network_watcher_flow_log_name = "my_nw_flow_log"
+
+  flow_log_enabled            = true
+  flow_log_logging_enabled    = true
+  flow_log_storage_account_id = "xxx"
```

Comment:
Please create a storage in the example and use its output to have a fully comprehensive example.

Reply:
done!